keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2641,42 +2771,36 @@ when the feature is disabled.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
+Workloads not using ResourceClaims should not be impacted because the new code
```

Comment:
Can't comment on the unchanged lines above - but I think the new feature gate needs to be added in the PRR section as well. We will likely need a fair bit of explanation about what happens when with that gate too - stuff that will eventually be in the new KEP for that gate, but needs to be in here for now.

Reply:
Added. The "Rollout, Upgrade and Rollback Planning" section is only needed for beta, so I left it out there. "Scalability" is "encouraged". I left it out there, too, for the sake of simplicity.

Reply:
k